skbio/stats/_subsample.py

```diff
@@ -231,7 +231,7 @@ def subsample_counts(counts, n, replace=False):
         raise ValueError("Only 1-D vectors are supported.")
 
     counts_sum = counts.sum()
-    if n > counts_sum:
+    if n > counts_sum and not replace:
```

Comment:
it looks like a pandas problem. it should be addressed separately. the previous merged PRs also failed to pass on those 2 python versions.